services/src/datasets/external/netcdfcf/schema/migration_0000_initial.sql

```diff
@@ -0,0 +1,248 @@
+-- VERSIONING
+
+CREATE TABLE geoengine (
+    database_version text NOT NULL
+);
+
+INSERT INTO geoengine (database_version) VALUES (
+    '0000_initial'
+);
+
+
+-- TYPE DEFINITIONS
+
+CREATE TYPE "RasterDataType" AS ENUM (
+    'U8',
+    'U16',
+    'U32',
+    'U64',
+    'I8',
+    'I16',
+    'I32',
+    'I64',
+    'F32',
+    'F64'
+);
+
+CREATE DOMAIN "RgbaColor" AS smallint [4] CHECK (
+    0 <= ALL(value) AND 255 >= ALL(value)
+);
+
+CREATE TYPE "Breakpoint" AS (
+    "value" double precision,
+    color "RgbaColor"
+);
+
+CREATE TYPE "ColorizerType" AS ENUM (
+    'LinearGradient', 'LogarithmicGradient', 'Palette', 'Rgba'
+);
+
+CREATE TYPE "Colorizer" AS (
```

Comment:
it's not great that we have to define the Colorizer redundantly. Especially if we change these types, we also have to migrate them in two places.

Would it make sense to require the user to put the netcdf schema into the same database as the geoengine schema? Then we could access them by "geoengine."Colorizer"".